systemtests/src/test/java/io/enmasse/systemtest/common/StandardPersistentMessagesTest.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2019, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.enmasse.systemtest.common;
```

Comment:
Shouldn't this be io.enmasse.systemtest.standard

Reply:
the same from above